File: 8_kyu/Grasshopper_Grade_book.rs
fn get_grade(s1: u16, s2: u16, s3: u16) -> char {
    let avg = (s1 + s2 + s3) / 3;
    for (limit, grade) in [(60, 'F'), (70, 'D'), (80, 'C'), (90, 'B')] {
        if avg < limit {
            return grade;
        }
    }
    'A'
}

File: 8_kyu/Grasshopper_Grade_book.py
def get_grade(s1: int, s2: int, s3: int) -> str:
    """
    Returns the grade associated with the avrerage of the scores.
    """
    mean = (s1+s2+s3)/3
    for limit,grade in ((90,'A'),(80,'B'),(70,'C'),(60,'D'),(0,'F')):
        if mean >= limit:
            return grade
    # negative mean value of the scores
    return None
